### PR TITLE
Add type="button" to toggle buttons...

### PR DESCRIPTION
**Pull request by franktopel**
... otherwise the according toggle buttons trigger a form submit if used inside a form element.

### CODE REVIEW

File: docs/4.0/components/collapse.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Using the [card]({{ site.baseurl }}/docs/{{ site.docs_version }}/components/card
   <div class="card">
     <div class="card-header" id="headingOne">
       <h5 class="mb-0">
-        <button class="btn btn-link" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
```
Comment (vsn4ik) [inline docs/4.0/components/collapse.md]: Already exists

**Comment by franktopel** [comment]
No, they definitely don't have it in the displayed sample code:

https://getbootstrap.com/docs/4.0/components/collapse/

grafik

**Comment by patrickhlauke** [comment]
the live documentation is older than the current state of the repo.

again, this is already fixed in the repo. if you look closely at your PR, you'll see you're trying to add a *second* `type="button"`

https://github.com/twbs/bootstrap/pull/25643

**Comment by patrickhlauke** [comment]
grafik

**Pull request closed without merging**